rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -0,0 +1,76 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name foreman_ansible
+
+%global foreman_dir /usr/share/foreman
+%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
+
+Summary: Ansible integration with Foreman (theforeman.org)
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.2.1
+Release: 1%{?foremandist}%{?dist}
+Group:   Applications/System
+License: GPLv3
+URL:     https://github.com/theforeman/foreman_ansible
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+Requires: foreman >= 1.6.0
+
+Requires: %{?scl_prefix_ruby}ruby(release)
+Requires: %{?scl_prefix_ruby}rubygems-devel
+Requires: %{?scl_prefix_ruby}rubygems
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+BuildRequires: %{?scl_prefix_ruby}rubygems
+BuildRequires: foreman-plugin >= 1.6.0
+
+BuildArch: noarch
+
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+Provides: foreman-plugin-ansible
+
+%description
+Ansible integration with Foreman.
+
+
+%package doc
+Summary: Documentation for %{name}
+Group: Documentation
+Requires: %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+
+%prep
+%setup -q -c -T
+%gem_install -n %{SOURCE0}
```

Comment:
missing an SCL wrapper here, so it's executing in a non-SCL environment I think.  Not sure why it happens to be installed and work on EL7, it might be that some of our packages are causing non-SCL Ruby to be installed too (http://projects.theforeman.org/issues/12915).

Change to:

<pre>
%{?scl:scl enable %{scl} - << \EOF}
%gem_install -n %{SOURCE0} 
%{?scl:EOF}
</pre>